### PR TITLE
DOCSP-43519 -- Add supported operating systems and architectures for local deployment of Atlas

### PR DESCRIPTION
**Pull request by jvincent-mongodb**
Add supported operating systems and architectures for local deployment of Atlas

NOTE: I've combined the required changes from https://jira.mongodb.org/browse/DOCSP-43126 with this PR. 

- [DOCSP-number](https://jira.mongodb.org/browse/DOCSP-43519)
- [STAGING](https://deploy-preview-676--docs-atlas-cli.netlify.app/atlas-cli-deploy-local/#supported-os-for-local-atlas-deployments)

- [LATEST BUILD LOG](paste the build here)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

**Comment by fmenezes** [comment]
Are you leaving the banner removal for another PR?

File: source/atlas-cli-deploy-local.txt [hunk 1]
```diff
@@ -21,10 +21,6 @@ create a local |service| deployment. In this tutorial, we will deploy a
 single-node replica set on your local computer. You can then manage 
 your deployment, and use |fts| and {+avs+}.
 
-You should use local deployments for testing and development only.
-
-.. include:: /includes/fact-atlas-deployments-preview.rst
-
```
Comment (tibulca) [inline source/atlas-cli-deploy-local.txt:27]: @jvincent-mongodb can you please remove it from all 4 places `source/atlas-cli-tutorials.txt`, `source/telemetry.txt`, `source/atlas-cli-local-cloud.txt`, `source/atlas-cli-deploy-local.txt` and probably delete `source/includes/fact-atlas-deployments-preview.rst` too.
Search I used: https://github.com/search?q=repo%3Amongodb%2Fdocs-atlas-cli%20fact-atlas-deployments-preview&type=code

cc @corryroot

File: source/atlas-cli-deploy-local.txt [hunk 2]
```diff
@@ -49,6 +45,24 @@ Supported OS for Local Atlas Deployments
      - 2
      - 2
 
+   * - Ubuntu
```
Comment (tibulca) [inline source/atlas-cli-deploy-local.txt:48]: at line 54/68 there is a note that has to be removed:
```
.. note::

   Windows and Ubuntu are not fully supported for local |service| 
   deployments during the Public Preview stage.
```

Comment (tibulca) [reply source/atlas-cli-deploy-local.txt:48]: I see the lines removed in the last commit, thank you :)

**Review by lmkerbey-mdb — APPROVED** [review]
LGTM.

**Review by fmenezes — APPROVED** [review]
LGTM

**Review by fmenezes — APPROVED** [review]

**Review by tibulca — APPROVED** [review]

**Comment by tibulca** [comment]
can you please remove this banner: https://github.com/mongodb/docs-atlas-cli/blob/fa356ab5f1e7db44598ac63d7eebfc15aa22f7e5/snooty.toml#L128
```
targets = ["command/atlas-deployments*.txt"]
variant = "warning"
value = """\
    **Public Preview.** \
    The ``atlas deployments`` command and subsequent related commands are available as a Public Preview. The feature and corresponding documentation may change at any time in the Public Preview stage. To ask questions and provide feedback, see the `Atlas CLI Local Development Community Forum <https://www.mongodb.com/community/forums/tag/local-dev-atlas-cli>`__.\
    """
```

**Pull request merged**